src/godot/api/udas.d

```diff
@@ -233,3 +233,19 @@ A UDA to mark a static field to be used by Godot as a global singleton object.
 struct Singleton {
 
 }
+
+/++ 
+Advanced attribute that tells GC to ignore this class when scanning memory for pointers.
+
+Note that GodotScript objects are not allocated by GC and 
+uses GC.addRange mechanism to tell GC to scan memory for pointers inside that object.
+This attribute has effect that any pointer to a GC allocated memory will become a non-owning reference,
+and will no longer prevent garbage collection despite that there might still be live references in it.
+Simply put, marked objects may have its GC managed data collected when you don't expect it.
```

Comment:
The terminology is inaccurate here, making it confusing IMO. It's not about telling the GC to ignore anything - were you to `new` the godot script it would be registered normally, attribute or no. What about:
```D
Advanced attribute that tells Godot memory allocation funtions to not register it to D garbage collector.
```

Also, an owning pointer as I understand it means something like Godot `Node`s or `std.typecons.Unique` - that is, when you delete them the pointee gets deleted too. This means normal D references are always non-owning (unless you enable ownership/borrowing and add `@live` to your function). I think the correct term here is "weak reference" .

Reply:
Well ok, the first line is indeed isn't clear enough.

But the rest is depends, I guess.
The comment explains what it does.
Technically the runtime isn't aware there is typed block of memory that may contain GC allocated memory, so by not letting it know about them is basically same as saying "do not scan" this.

This sort of "weak reference" is basically useless if there is no way to check its validity (there isn't), which is why we had such confusing bugs that looks like memory corruption.